src/main/java/uk/gov/laa/ccms/caab/bean/ActiveCase.java

```diff
@@ -12,6 +12,8 @@ public class ActiveCase {
   private String client;
   private String clientReferenceNumber;
 
+  private Integer applicationId;
   private String caseReferenceNumber;
   private String providerCaseReferenceNumber;
+  private Integer providerId;
```

Comment:
Is there a reason we cant just use the id thats in the userDetails?

Reply:
I wasn't convinced that it would always be the same id. userDetails holds the primary providerId for the user. 
But they can have links to others - eg. chambers workers have access to multiple providers.